dash-renderer/src/checkPropTypes.js

```diff
@@ -21,7 +21,7 @@ export default function checkPropTypes(
     values,
     location,
     componentName,
-    getStack
+    getStack = null
```

Comment:
?

Reply:
When TS code calls JS code it tries to figure out what's going on anyway - without a default value on the parameter it thinks it's mandatory. Could make it `undefined` instead.